kolibri/auth/models.py

```diff
@@ -495,6 +575,18 @@ def filter_readable(self, queryset):
     def __str__(self):
         return self.get_full_name() or self.username
 
+    def has_perm(self, perm, obj=None):
+        # ensure the DeviceOwner has full access to the Django admin
+        return True
+
+    def has_perms(self, perm_list, obj=None):
+        # ensure the DeviceOwner has full access to the Django admin
+        return True
+
+    def has_module_perms(self, app_label):
+        # ensure the DeviceOwner has full access to the Django admin
+        return True
+
```

Comment:
These are needed in order to allow a logged-in `DeviceOwner` to access the Django admin, as a superuser.

(It's unlike we would be directing users there under normal circumstances, but it could be helpful for admins in the process of debugging something)